src/System-Settings-Browser/SettingBrowser.class.st

```diff
@@ -292,12 +281,6 @@ SettingBrowser class >> taskbarIconName [
 	^#smallConfiguration
 ]
 
-{ #category : #'private helpers' }
-SettingBrowser class >> viewAllSettings [
-	"self viewAllSettings"
-	Smalltalk tools workspace openContents: self allSettingsState label: 'Current Setting assignments'
-]
-
 { #category : #'help system' }
 SettingBrowser class >> wikiStyleHelp [
 	<wikiStyleHelp: #'The setting browser'>
```

Comment:
This pragma is not used anymore

Reply:
this is just code that github shows around the diff... not related to this change at all and thus should not be done as part of this PR